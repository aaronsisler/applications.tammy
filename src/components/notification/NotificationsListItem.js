import React from 'react';
import { connect } from 'react-redux';
import PropTypes from 'prop-types';

export class NotificationsListItem extends React.Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div className="notifications_list_item">
                <div className="notifications_list_item__message">
                    {this.props.notificationMessage}
                </div>
            </div>
        );
    }
}

/* istanbul ignore next */
const mapDispatchToProps = () => ({
});

export default connect(undefined, mapDispatchToProps)(NotificationsListItem);

NotificationsListItem.propTypes = {
    notificationMessage: PropTypes.string.isRequired,
};
