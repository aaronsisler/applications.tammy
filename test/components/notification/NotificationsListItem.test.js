import React from 'react';
import { shallow } from 'enzyme';
import { NotificationsListItem } from 'Notification/NotificationsListItem';

describe('NotificationsListItem', () => {
    let wrapper;

    const buildWrapper = () => {
        wrapper = shallow(
            <NotificationsListItem
                notificationMessage={'I am a notification message'}
            />);
    }

    it('should render NotificationsIcon correctly', () => {
        buildWrapper();
        expect(wrapper).toMatchSnapshot();
    });
});
