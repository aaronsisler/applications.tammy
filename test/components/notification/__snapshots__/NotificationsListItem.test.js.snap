// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationsListItem should render NotificationsIcon correctly 1`] = `
<div
  className="notifications_list_item"
>
  <div
    className="notifications_list_item__message"
  >
    I am a notification message
  </div>
</div>
`;
